packages/rx_bloc_cli/example/README.md

```diff
@@ -54,31 +64,41 @@ You define localizations by adding a translation file in the `lib/l10n/arb/app_[
 
+<div id="analytics"/>
+
 ### Analytics
 
 [Firebase analytics][firebase_analytics_lnk] track how your app is used. Analytics are available for iOS, Android and Web and support flavors.
 
 Before you start using analytics, you need to add platform specific configurations:
-1. The `iOS` configuration files can be found at `ios/Runner/Firebase/[flavor]/GoogleService-Info.plist`
+1. The `iOS` configuration files can be found at `ios/environments/[flavor]]/firebase/GoogleService-Info.plist`
 2. For `Android` the configuration files are located at `android/app/src/[flavor]/google-services.json`
 3. All `Web` analytics configurations can be found inside `lib/base/app/config/firebase_web_config.js`
 
 Every flavor represents a separate Firebase project that will be used for app tracking. For each flavor, based on the targeted platforms you'll have to download the [configuration files][firebase_configs_lnk] and place them in the appropriate location mentioned above.
 
+*Note*: When ran as `development` flavor, `.dev` is appended to the package name. Likewise, `.stag` is appended to the package name when using `staging` flavor. If using separate analytics for different flavors, make sure you specify the full package name with the correct extension (for instance: `com.companyname.projectname.dev` for the `dev` environment).
+
+<div id="httpClient"/>
+
 ### Http client
 
 Your project has integrated HTTP-client, using [dio][dio_lnk] and [retrofit][retrofit_lnk]. That helps you to easily communicate with APIs and support Interceptors, Global configuration, FormData, Request Cancellation, File downloading, Timeout etc.
 
-To use its benefits you should define a data model in `lib/base/models/`, using [json_annotation][json_annotation_lnk] and [json_serializable][json_serializable_lnk]. Define your remote data source in folder `lib/base/data_sources/remote/` with methods and real Url, using [retrofit][retrofit_lnk]. In your dependencies class(in this case lib\feature_counter\di\counter_dependencies.dart) specify which data source you are going to use in every repository.
+To use its benefits you should define a data model in `lib/base/models/`, using [json_annotation][json_annotation_lnk] and [json_serializable][json_serializable_lnk]. Define your remote data source in folder `lib/base/data_sources/remote/` with methods and real Url, using [retrofit][retrofit_lnk]. In your dependencies class (in this case `lib/feature_counter/di/counter_dependencies.dart` ) specify which data source you are going to use in every repository.
+
+In our project in directory `bin/server/` we have implemented local Rest API using [shelf][shelf_lnk]. To test the project you have to run it first. In your root folder run `bin/start_server.sh`. If you are running in your emulator and can not connect to the server, check this [solution][solution_lnk].
 
```

Comment:
I think that those rows are redundant and we don't need them anymore. Now we have description for the server itself. You only may move the last link there or specify in the description where to run the command 'adb reverse tcp:80800 tcp:80800' .

Reply:
Fixed!